### DISCUSSION TITLE
Add CTIM satellite

### DISCUSSION THREAD

**Discussion by ArgoNavi** [dernasherbrezon/r2cloud#125] [Q&A] [answered]
Could you add support for CTIM (Compact Total Irradiance Monitor) to r2cloud?

https://db.satnogs.org/satellite/CSGC-5605-1718-3109-5402

**Comment by dernasherbrezon** [comment, marked as answer]
Should be fixed:
https://github.com/dernasherbrezon/r2cloud/commit/30ba51ddec3e5e3b68d105e4e1ee7a9df9c71d15
